src/lib/pages/backoffice/Document/DocumentDetails/DocumentDetails.js

```diff
@@ -104,8 +111,10 @@ DocumentDetails.propTypes = {
       documentPid: PropTypes.string,
     }),
   }).isRequired,
+  renderTabTitle: PropTypes.func,
```

Comment:
if you add as required prop, we can avoid the if statement on each function call